README.md

```diff
@@ -52,3 +52,12 @@ ${STACK}/start.sh
 ```bash
 ${STACK}/stop.sh
 ```
+
+# Confluent Cloud
+
+If you are using Confluent Cloud, you can also use the [Metrics API](https://docs.confluent.io/current/cloud/metrics-api.html), a queryable HTTP API to pull operational metrics about your Confluent Cloud deployment.
+See example usage:
+
+* https://github.com/Dabz/ccloudexporter
+* https://github.com/vdesabou/kafka-docker-playground/tree/master/ccloud/ccloudexporter
+* https://github.com/derplarsen/ccloud_utilization_analyzer
```

Comment:
Would be interesting to check if 2nd and 3rd links are really different from examples in Dabz/ccloudexporter. AFAICS dashboards look quite similar and it might be enough linking the first one.

Reply:
cc: @Dabz @vdesabou @derplarsen for feedback

Reply:
I'll leave as-is for now.  Folks can prune/optimize in a separate PR